src/Bridges/HttpDI/HttpExtension.php

```diff
@@ -79,10 +81,25 @@ public function afterCompile(Nette\PhpGenerator\ClassType $class)
 			} elseif (preg_match('#^https?:#', $frames)) {
 				$frames = "ALLOW-FROM $frames";
 			}
-			$initialize->addBody('$this->getService(?)->setHeader(?, ?);', [$this->prefix('response'), 'X-Frame-Options', $frames]);
+			$headers['X-Frame-Options'] = $frames;
 		}
 
-		foreach ($config['headers'] as $key => $value) {
+		$csp = & $headers['Content-Security-Policy'];
+		foreach ($config['policy'] as $type => $policy) {
```

Comment:
Is it a good idea to have it as this much general key? wouldn't `csp` be better?